packages/getting-started/src/browser/getting-started-widget.tsx

```diff
@@ -109,17 +109,17 @@ export class GettingStartedWidget extends ReactWidget {
     }
 
     protected renderOpen(): React.ReactNode {
+        const requireSingleOpen = isOSX || !environment.electron.is();
+        const open = (requireSingleOpen) ? <a href='#' onClick={this.doOpen}>Open</a> : '';
+        const openFile = (!requireSingleOpen) ? <a href='#' onClick={this.doOpenFile}>Open File</a> : '';
+        const openFolder = (!requireSingleOpen) ? <a href='#' onClick={this.doOpenFolder}>Open Folder</a> : '';
+        const openWorkspace = <a href='#' onClick={this.doOpenWorkspace}>Open Workspace</a>;
         return <div className='gs-section'>
-            <h3 className='gs-section-header'>
-                <i className='fa fa-folder-open'></i>
-                Open
-            </h3>
-            <div className='gs-action-container'>
-                <a href='#' onClick={this.doOpen}>Open</a>
-            </div>
-            <div className='gs-action-container'>
-                <a href='#' onClick={this.doOpenWorkspace}>Open Workspace</a>
-            </div>
+            <h3 className='gs-section-header'><i className='fa fa-folder-open'></i>Open</h3>
+            <div className='gs-action-container'>{open}</div>
+            <div className='gs-action-container'>{openFile}</div>
+            <div className='gs-action-container'>{openFolder}</div>
+            <div className='gs-action-container'>{openWorkspace}</div>
```

Comment:
Will all the containers still be outputed even though the content is empty? Maybe it is not important, or React takes care of it?